backend/routes/twoRandomToppings.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express'
+import expandToppingsCategories from '../expandToppingsCategories'
+import { getToppingsCollection } from '../db'
+import { SuccessResponse } from '../response'
+import { Topping, ToppingWithCategoryId } from '../topping'
+
+type TwoRandomToppingsResponse = SuccessResponse<[Topping,Topping]>
+
+export default async function twoRandomToppings(
+  request: Request, response: Response<TwoRandomToppingsResponse>
+) {
+  const exclude = request.query.exclude as (string | null)
+
+  const toppingsCollection = await getToppingsCollection()
+  let toppings: null | [ToppingWithCategoryId,ToppingWithCategoryId] = null
+
+  const forbiddenToppings =
+    (toppings: ToppingWithCategoryId[]) => (
+      exclude &&
+        toppings.some(topping =>
+          exclude === topping._id.toString())
```

Comment:
what is .some doing?

Reply:
it means "is there anything in `toppings` that makes this function return true?"

Reply:
👍